docs/src/meshes.md

```diff
@@ -60,3 +60,47 @@ mesh = SimpleMesh(points, connec)
 ```@example meshes
 viz(mesh, showfacets = true)
 ```
+
+```@example meshes
+# Convert topology to half edge topology for fast lookup
+ht = convert(HalfEdgeTopology, topology(mesh))
+# We can create the boundary facets of the first Ngon.
+# The boundary is parametrized on the dimension we are mapping from,
+# and the lower dimension we are mapping to.
+# The Ngons are of order 2 and are made up of segments with order 1:
+b = Boundary{2,1}(ht)
```

Comment:
I would use the symbol `\partial21` for the boundary relation so that it is explicit in the variable name.